src/util-streaming-buffer.c

```diff
@@ -1232,7 +1249,8 @@ static StreamingBufferRegion *BufferInsertAtRegionConsolidate(StreamingBuffer *s
 
     // 2. resize dst
     const uint32_t old_size = dst->buf_size;
-    const uint32_t dst_copy_offset = dst->stream_offset - dst_offset;
+    DEBUG_VALIDATE_BUG_ON(dst->stream_offset - dst_offset > UINT32_MAX);
```

Comment:
Same question as above. Actually in all the following calculations in this commit involving subtraction.

Reply:
Maybe this one is worth it... @victorjulien would you know ?